quickwit/quickwit-serve/src/ingest_api/rest_handler.rs

```diff
@@ -498,25 +500,6 @@ mod tests {
         universe.assert_quit().await;
     }
 
-    #[tokio::test]
-    async fn test_ingest_api_bulk_request_returns_400_if_malformed_source() {
```

Comment:
nitpick: we could keep a test on a malformed action

Reply:
Good point.